.github/workflows/build-readme.yml

```diff
@@ -0,0 +1,44 @@
+﻿name: build-readme
+on: workflow_dispatch
```

Comment:
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
https://docs.github.com/zh/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch